scripts/variantstore/variant_annotations_table/GvsValidateVAT.wdl

```diff
@@ -801,6 +811,90 @@ task SubpopulationAlleleNumber {
     }
 }
 
+task DuplicateAnnotations {
+    input {
+        String query_project_id
+        String fq_vat_table
+        String last_modified_timestamp
+    }
+    # there are vids with duplicate annotations
+    # make sure there aren't matching sites with mis-matched ANs and matching variants with mis-matched ACs
+
+    String pf_file = "pf.txt"
+    String results_file = "results.txt"
+
+    command <<<
+        set -e
+
+        echo "project_id = ~{query_project_id}" > ~/.bigqueryrc
+
+        bq query --nouse_legacy_sql --project_id=~{query_project_id} --format=csv 'SELECT * from
+        (SELECT contig, position, gvs_all_an, COUNT(DISTINCT gvs_all_an) AS an_count FROM `~{fq_vat_table}`
+        group by contig, position, gvs_all_an)
+        where  an_count >1
+        ' > bq_an_output.csv
```

Comment:
I think this could be simplifed with `HAVING`

```suggestion
        bq query --nouse_legacy_sql --project_id=~{query_project_id} --format=csv '

        SELECT contig, position, gvs_all_an, COUNT(*) FROM `~{fq_vat_table}`
        GROUP BY contig, position, gvs_all_an HAVING COUNT(*) > 1

        ' > bq_an_output.csv
```